src/main/java/net/imglib2/algorithm/morphology/TopHat.java

```diff
@@ -78,7 +78,8 @@ public class TopHat
 	 * decomposition. Each shape is processed in order as given in the list. If
 	 * the list is empty, the source image is returned.
 	 * <p>
-	 * 
+	 *
+	 * @implNote op names='morphology.topHat, morphology.whiteTopHat', type=Function
```

Comment:
Let's drop the `morphology.topHat` alias, in favor of `whiteTopHat` always, for symmetry with `blackTopHat`. That's what scikit-image does.

Reply:
Heh, SciJava Ops Image currently *only* uses `morphology.topHat`. I agree that symmetry would be good, but I'm also tempted to leave in the aliases. Maybe we can deprecate that soon?